src/main/java/org/spongepowered/api/CatalogType.java

```diff
@@ -49,8 +49,19 @@
      * default/vanilla minecraft types.</p>
      *
      * @return The unique identifier of this dummy type
+     * @deprecated use {@link #getKey()}
      */
-    String getId();
+    @Deprecated
+    default String getId() {
+        return this.getKey().toString();
+    }
+
+    /**
+     * Gets the catalog key for this catalog type.
```

Comment:
Maybe explain that the catalog key is provided and should be used to retrieve this `CatalogType` from a `CatalogTypeRegistry`.

Reply:
And mention that this key should be used for serialization.